lisp_ease/lisp-ease.asd

```diff
@@ -0,0 +1 @@
+src/lisp-ease.asd
```

Comment:
Please move *.asd file up to the root of the ROS package. Then use module "src" to refer to the files. Please rename "queries" directory into "src".